cifar/resnet.py

```diff
@@ -0,0 +1,135 @@
+"""
+Implementation of ResNets for CIFAR-10 as per the original paper [https://arxiv.org/abs/1512.03385].
+Configurations include ResNet-20, ResNet-32, ResNet-44, ResNet-56, ResNet-110, ResNet-1202.
+
+There's no BatchNorm is mlx==0.0.4, using LayerNorm instead.
+
+Authors:
+    Sarthak Yadav, 2023
```

Comment:
Hey Sarthak, I really want to make sure you get credit for this, but I think we should avoid putting individual authors in code files as that will be really difficult to maintain (and also hard to make fair to future developers who edit and improve this example).

For now could you please remove this?

I will discuss with the team about how we can best assign credit to our many contributors and then follow up and make sure to get your name added where it makes sense!

Reply:
> For now could you please remove this?

Of course, that's not a problem. Out of habit more than anything, this was the only file I wrote a header for 😅. There usually isn't an "established" way to do this between open-source frameworks (SpeechBrain does it this way) and that's fine!